examples_01/01_rot13_server_libevent.c

```diff
@@ -93,21 +92,23 @@ do_read(evutil_socket_t fd, short events, void *arg)
         for (i=0; i < result; ++i)  {
             if (state->buffer_used < sizeof(state->buffer))
                 state->buffer[state->buffer_used++] = rot13_char(buf[i]);
-            if (buf[i] == '\n') {
```

Comment:
This is incorrect, this makes echo server non echo, and will write response only when the client will close the connection